api/src/wfl/util.clj

```diff
@@ -503,3 +493,9 @@
      (columns-rows->tsv [(format-entity-type columns)] rows file)))
   ([tsv-type columns rows]
    (str (columns-rows->terra-tsv tsv-type columns rows (StringWriter.)))))
+
+(defn utc-now
+  []
+  (-> (ZoneId/of "UTC")
+      (OffsetDateTime/now)
+      (.format (DateTimeFormatter/ofPattern "yyyy-MM-dd HH:mm:ss"))))
```

Comment:
I think this will cause problems.
First we don't know what time zone the dataset is using for its `run-date` column.
Second, big query needs a Timestamp in the form `yyyy-MM-ddTHH:mm:ss` - this way your queries only have date precision.

Reply:
Per offline discussion, we need to:
- Ask the working group why this `run_date` field is not a TIMESTAMP but a STRING
- Make sure it's UTC in any cases
- Add a `T` here.

Reply:
Moved (and broke) this into the covid module, based on the latest info, `run_date` field is now a TIMESTAMP! Let's talk about this again.